test/integration/noderesourcetopology_test.go

```diff
@@ -700,10 +944,10 @@ func cleanupNodeResourceTopologies(ctx context.Context, topologyClient *versione
 	}
 }
 
-func withReqAndLimitHelper(resMap map[v1.ResourceName]string, cList []v1.Container, prefix string) []v1.Container {
+func withReqAndLimitHelper(resMap map[string]string, cList []v1.Container, prefix string) []v1.Container {
```

Comment:
This way we can use a shorter and more readable version for initializing the tests:
using:

```
cntReq: []map[string]string{
				{cpu: "20", memory: "40G"},
				{cpu: "20", memory: "40G"},
				{cpu: "20", memory: "10G"},
			},
```

instead of:

```
cntReq: []map[v1.ResourceName]string{
				{v1.ResourceCPU: "20", v1.ResourceMemory: "40G"},
				{v1.ResourceCPU: "20", v1.ResourceMemory: "40G"},
				{v1.ResourceCPU: "20", v1.ResourceMemory: "10G"},
			},
```
Eventually,  ```withReqAndLimitHelper``` gets ```cntReq```

Reply:
Ack, Thanks for the clarification.